02-electric/electric-section-three/electric-section-three.css

```diff
@@ -1,65 +1,327 @@
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: var(--container-padding-mobile);
   
-    .electric-section-three-subcontainer > *{
-          border: solid  1px black;
-      }
-      
-      .electric-section-three-img-container{
-          grid-column: 1/2;
-          
-      }
-
-    .electric-section-three-img{
-      display: block;
-        width: 100%;
-    }
-      
-    .electric-section-three-title{
-        grid-column: 1/2;
-         text-align: center;
-      }
-
-    .electric-section-three-content{
-          grid-column: 1/2;
-         text-align: justify;
-      }
-      
-      .electric-section-three-small-content-container{
-        grid-column: 1/2;
-        display: grid;
-        grid-template-columns: 1fr 3fr;
-      }
-      .electric-section-three-small-img{
-        grid-column: 1/2;
-        justify-self: center;
-        align-self: center;
-        width: 100%;
-      }
-      
-      .electric-section-three-small-prg{
-        grid-column:2/3;
-
-      }
-
-    .electric-section-three-btn{
-        grid-column: 1/2;
-        width:90px;
-        height: 40px ;
-        justify-self: center;
-      }
\ No newline at end of file
+}
+
+@media (min-width:500px) {
+  .electric-section-three-container {
+    padding: var(--container-padding-desktop);
+  }
+}
+
+.electric-section-three-subcontainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+  "electric-section-three-title"
+  "electric-section-three-part-one-container"
+  "electric-section-three-part-two-container"
+  "electric-section-three-part-three-container";
+  justify-content: center;
+  row-gap: var(--subsection-container-row-gap-mobile);
+  column-gap:none;
+  padding: var(--subcontainer-padding-mobile);
+  border: solid 1px black;
+  background-color: var(--level1-background-color);
+}
+
+@media (min-width:500px){
+  .electric-section-three-subcontainer{
+    grid-template-columns: repeat(2,1fr);
+    grid-template-areas:
+    "electric-section-three-title electric-section-three-title"
+    "electric-section-three-part-one-container electric-section-three-part-one-container"
+    "electric-section-three-part-two-container electric-section-three-part-three-container";
+    padding: var(--subcontainer-padding-desktop);
+    row-gap: var(--subsection-container-row-gap-desktop);
+    column-gap:var(--subsection-container-column-gap-desktop) ;
+  }
+}
+
+.electric-section-three-title {
+  grid-area:electric-section-three-title;
+  text-align: center;
+  font-size: var(--title-font-size-mobile);
+  font-weight: var(--title-font-weight-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-title {
+    font-size: var(--title-font-size-desktop);
+    font-weight: var(--title-font-weight-desktop);
+  }
+}
+
+/*****PART one******/
+
+.electric-section-three-part-one-container{
+  grid-area:electric-section-three-part-one-container;
+  display: grid;
+  grid-template-columns: 1fr;
```

Comment:
please add 

align-content: start;

to all three of the  part-one /  part-two /  part-three containers

(because the images are not sitting at the top and are being stretched vertically)

Reply:
done

Reply:
i think some of image container have been stretched because the parent container is stretched, but any way i added, its good to have it to control it more